Packs/FeedAutofocus/Integrations/FeedAutofocus/FeedAutofocus.py

```diff
@@ -21,28 +22,52 @@ class Client(BaseClient):
         insecure(bool): Use SSH on http request.
         proxy(str): Use system proxy.
         indicator_feeds(List): A list of indicator feed types to bring from AutoFocus.
-        output_feed_id(str): The ID given to the custom feed.
-        output_feed_name(str): The name given to the custom feed.
+        custom_feed_urls(str): The URLs of the custom feeds to fetch.
     """
 
-    def __init__(self, api_key, insecure, proxy, indicator_feeds, output_feed_id="", output_feed_name=""):
-        self.url_feed_base_url = "https://autofocus.paloaltonetworks.com/api/v1.0/IOCFeed/"
+    def __init__(self, api_key, insecure, proxy, indicator_feeds, custom_feed_urls=None):
         self.daily_feed_base_url = "https://autofocus.paloaltonetworks.com/api/v1.0/output/threatFeedResult"
         self.headers = {
             "apiKey": api_key,
             'Content-Type': "application/json"
         }
         self.indicator_feeds = indicator_feeds
-        if 'URL Feed' in indicator_feeds and (output_feed_name is None or output_feed_id is None):
+        if 'Custom Feed' in indicator_feeds and custom_feed_urls is None:
```

Comment:
this could be '' and it will pass, need to fix this

Reply:
done